pkg/identity/numericidentity.go

```diff
@@ -365,6 +373,23 @@ func (id NumericIdentity) Uint32() uint32 {
 	return uint32(id)
 }
 
+// GetLocalNodeID returns the configured local node numeric identity that is
+// set in tunnel headers when encapsulating packets originating from the local
+// node.
+func GetLocalNodeID() NumericIdentity {
+	localNodeMutex.Lock()
+	defer localNodeMutex.Unlock()
+	return localNodeIdentity
```

Comment:
Given the above change:
```suggestion
	localNodeIdentity.Lock()
	defer localNodeIdentity.Unlock()
	return localNodeIdentity.identity
```

Reply:
Thx.